### PR TITLE
[KYUUBI #3170] Expose thrift binary connection metrics

### PR DESCRIPTION
**Pull request by lightning-L**
### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-metrics/src/main/scala/org/apache/kyuubi/metrics/MetricsConstants.scala [hunk 1]
```diff
@@ -32,6 +32,7 @@ object MetricsConstants {
 
   final private val CONN = KYUUBI + "connection."
   final private val THRIFT_HTTP_CONN = KYUUBI + "thrift.http.connection."
+  final private val BINARY_FRONTEND_CONNECTION = KYUUBI + "binary.frontend.connection."
```
Comment (pan3793) [inline kyuubi-metrics/src/main/scala/org/apache/kyuubi/metrics/MetricsConstants.scala:35]:
```suggestion
final private val THRIFT_BINARY_CONN = KYUUBI + "thrift.binary.connection."
```

**Review by pan3793 — APPROVED** [review]

**Comment by lightning-L** [comment]
@turboFei

**Comment by lightning-L** [comment]
reference: https://github.com/apache/hive/blob/9ad5a0b28b136521ded27b4b5418f61b610da788/service/src/java/org/apache/hive/service/cli/thrift/ThriftBinaryCLIService.java

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**